### DISCUSSION TITLE
Seeedstudio 7-Inch Smart Display

### DISCUSSION THREAD

**Discussion by webbbn** [linux-chenxing/linux-chenxing.org#23] [General]
I noticed this up and coming device on Seeedstudio: https://www.seeedstudio.com/SigmastarSSD202-7-inch-Smart-Display-p-4885.html, which, through the magic of Google, brought me here.

This looks like a really interesting piece of hardware, but I'm concerned about the maturity of the OS support. It looks like the development is very active currently. Is it likely that this display will be at least marginally useful in the short to medium term?

**Comment by fifteenhex** [comment]
Hi,

This board seems to be the IDO-SBC2D70 which is available in other places like taobao for ~$70.
I have one of these on it's way to me at the moment. I have a bunch of the SoM it's based on and have those running mainline.

There is also a really cool looking board with a 4 inch display coming out: https://www.alibaba.com/product-detail/square-4inch-LCD-panel-based-on_1600205049030.html?spm=a2756.order-detail-ta-bn-b.0.0.3f8f2fc2hIkMU2

Mainline support for the display really needs someone that knows how to write those sorts of drivers to help me out.
I think I know where to get the information we need (there is code to drive displays in the vendor u-boot) but it will take me a while to learn how to write the drivers and then write the drivers. :)

I'm taking apart the u-boot code and filling in register offsets here:
http://linux-chenxing.org/ip/display.html
I'm hoping someone that knows what they are doing will come along and help out eventually. ;)

Thanks,

Daniel
